frontend/app/scripts/views/layouts/home_layout_view.js

```diff
@@ -66,22 +44,26 @@ define([
         .html(navigationView.render().el);
     },
 
-    bindCollectionEvents: function(options) {
-      var collection = options.collection,
-          $el        = options.$el,
-          that       = this;
-      collection.on('sync', function(collection) {
-        that.renderCarsListView($el, collection.toJSON());
+    bindCarsEvents: function() {
+      var that = this;
+      this.cars.on('sync', function() {
+        that.renderCarsListView(that.cars.toJSON());
       });
-      collection.on('error', function() {
-        console.log('render error');
+      this.cars.on('error', function() {
+        that.renderOnErrorMessage();
       });
     },
 
-    renderCarsListView: function($el, cars) {
+    renderCarsListView: function(cars) {
       var carsListView = new CarsListView({ cars: cars });
       this.subViews.push(carsListView);
-      $el.html(carsListView.render().el);
+      this.$el.find('#car-list-view')
+        .html(carsListView.render().el);
+    },
+
+    renderOnErrorMessage: function() {
```

Comment:
Render a view which prints something when there is an error while fetching cars. This will help to keep a more cohesive responsibility of a layout: delegate actions to other views.